### ISSUE TITLE
Unit is malformed: no such unit

### ISSUE THREAD

**Issue by coder543** [tiffany352/rink-rs#115]
Within the last several days, n several of my machines (all that I've tested), `rink` has started outputting this every time I start it:

```
$ rink
Unit копеика is malformed: No such unit RUB, did you mean ruby?
Unit kopek is malformed: No such unit копеика
Unit rouble is malformed: No such unit RUB, did you mean ruby?
Unit ruble is malformed: No such unit RUB, did you mean ruby?
Unit копеики is malformed: No such unit копеика
Unit рубль is malformed: No such unit RUB, did you mean ruby?
Unit ₽ is malformed: No such unit RUB, did you mean ruby?
> 
```

I haven't had time to dig in and understand what's happening, but I'm assuming some upstream currency format changed in an incompatible way.

**Comment by tiffany352** [comment]
It looks like the European Central Bank deleted the Ruble from their currency data. I can probably fix this on the backend side by including a hardcoded value for Ruble <-> Euro.

**Comment by tiffany352** [comment]
Fixed in f10a39c01e8332948f1be367c559904f4ffb70f3 which should automatically roll out to everyone using 0.6+ (since it fetches the data from https://rinkcalc.app/data/currency.json by default). I actually added default values for all ECB currencies so this exact failure shouldn't happen again.

Data for the Ruble won't live update until I can find an API for it. A lot of the ones I've seen are paid access only.
```
❯ ruble
Definition: RUB = (1 / 141.951) EUR = approx. 7.044684 millieuro (money; EUR). Fetching live data failed. Fallback value provided from xe.com on 2022-03-09.
```
